solarwinds_apm/configurator.py

```diff
@@ -116,7 +116,11 @@ def _configure_otel_components(
         if apm_config.agent_enabled:
             # set MeterProvider first via metrics_exporter config
             self._configure_metrics_exporter(apm_config)
+            # This processor only defines on_start
```

Comment:
I added this and other comments because I keep forgetting the reasons for the processors registration order.